unit_tests/speech/test_client.py

```diff
@@ -172,6 +172,24 @@ def test_sync_recognize_with_empty_results(self):
             client.sync_recognize(None, self.AUDIO_SOURCE_URI, Encoding.FLAC,
                                   self.SAMPLE_RATE)
 
+    def test_async_recognize(self):
+        from unit_tests.speech._fixtures import ASYNC_RECOGNIZE_RESPONSE
+        from google.cloud.speech.encoding import Encoding
+        from google.cloud.speech.operation import Operation
+        RETURNED = ASYNC_RECOGNIZE_RESPONSE
+
+        credentials = _Credentials()
+        client = self._makeOne(credentials=credentials)
+        client.connection = _Connection(RETURNED)
+
+        encoding = Encoding.FLAC
+
+        operation = client.async_recognize(None, self.AUDIO_SOURCE_URI,
+                                           encoding,
+                                           self.SAMPLE_RATE)
+        self.assertIsInstance(operation, Operation)
+        self.assertFalse(operation.complete)
```

Comment:
I could make a `SpeechMetadata` class if you want. `operation.metadata.last_updated`..etc?